projects/storefrontlib/cms-components/myaccount/consent-management/components/consent-management.component.ts

```diff
@@ -95,11 +97,10 @@ export class ConsentManagementComponent implements OnInit, OnDestroy {
         }
       }),
       map(([templateList, anonymousTemplates]) => {
+        this.requiredConsents = this?.consentManagementService
```

Comment:
nitpick: `this` is defined

```suggestion
        this.requiredConsents = this.consentManagementService
```

Reply:
done